src/WebJobs.Script/Rpc/RpcInitializationService.cs

```diff
@@ -116,12 +127,18 @@ internal async Task InitializeRpcServerAsync()
 
         internal Task InitializeChannelsAsync()
         {
-            if (ShouldStartInPlaceholderMode())
+            if (StartAsPlaceholderPool())
```

Comment:
instead of adding another method, move this logic inside, ShouldStartInPlaceholderMode

Reply:
@pragnagopa -  I think we want to keep it as separate logic because we'd only like to start the relevant specific process?